### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Sneezry** [Authenticator-Extension/Authenticator#256]
- [x] High contrast theme
- [x] TTS support for code

High contrast theme has been ready in dev branch

![snipaste20181226_162109](https://user-images.githubusercontent.com/1621293/50438767-6805d580-092a-11e9-83f6-fd5c2508ef17.png)

TTS support has been ready on [`tts` branch](https://github.com/Authenticator-Extension/Authenticator/tree/tts).

I haven't decided if TTS feature should be merged into dev branch, yet.

TTS is not available for optional permission, which means we have to expand the permission scope to enable TTS. TTS API is only available in Chrome currently.

![snipaste20181226_144938](https://user-images.githubusercontent.com/1621293/50438603-a0f17a80-0929-11e9-9cec-df7f8c3e1b7c.png)

Also, I doubt if TTS support is necessary. Screen reader may be enough.

**Comment by Sneezry** [comment]
Let's wait for more feedback about TTS. Currently TTS will not be merged into dev branch.
